pom.xml

```diff
@@ -13,13 +13,26 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>8</source>
+                    <target>8</target>
```

Comment:
are we using JDK8 on local machine? anyway the version should correspond the actual Java version

Reply:
I am using java 8